tests/Unit/Execution/Utils/GlobalIdTest.php

```diff
@@ -12,8 +12,8 @@ class GlobalIdTest extends TestCase
      */
     public function itCanHandleGlobalIds(): void
     {
-        $globalId = GlobalId::encode('User', 'asdf');
-        $idParts = GlobalId::decode($globalId);
+        $globalId = app(GlobalId::class)->encode('User', 'asdf');
```

Comment:
Let's just `new` up an instance here, since this test is about the concrete implementation.